webpack/main.ts

```diff
@@ -54,8 +58,8 @@ configs.push((): webpack.Configuration => {
         cwd: __dirname,
         exclude: /node_modules/,
         failOnError: true,
-      }),
-    ].filter(Boolean),
+      }) as unknown as WebpackPluginInstance,
```

Comment:
Why need to cast as `WebpackPluginInstance`? _(looks weird within whole config)_

Reply:
Because the typing of `CircularDependencyPlugin` is incorrect and seems to be for an older version of webpack. It still works though...